locales/en/yourloops.json

```diff
@@ -56,6 +56,9 @@
   "breadcrumb-notifications": "Notifications",
   "browse": "Browse",
   "button-accept": "Accept",
+  "training-body": "New <trainingLink>{{training}}</trainingLink> available, please read what's new before continuing on yourloops. You can also read it later, by clicking in the footer link.",
+  "training-materials": "training materials",
```

Comment:
Translations start with an uppercase

Reply:
why ? It's embedded in the middle of a sentence ...